### PR TITLE
pidgin-mra: init at 54b2992

### PR DESCRIPTION
**Pull request by akaWolf**
Added Mail.ru Agent plugin for Pidgin / libpurple.

### CODE REVIEW

File: pkgs/applications/networking/instant-messengers/pidgin-plugins/pidgin-mra/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    rev = "${version}";
+    sha256 = "1nhfx9gi5lhh2xjr9rw600bb53ly2nwiqq422vc0f297qkm1q9y0";
+  };
+
+  buildInputs = [ pidgin ];
+
+  preConfigurePhases = "preConfigurePhase";
+
+  preConfigurePhase = ''
+    sed -i 's|-I/usr/include/libpurple|$(shell pkg-config --cflags purple)|' Makefile
```
Comment (vcunat) [inline pkgs/applications/networking/instant-messengers/pidgin-plugins/pidgin-mra/default.nix]: Is this line needed? The builder doesn't even have access to pkg-config and the build commands look fine already.

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -12336,6 +12336,8 @@ let
 
   pidginmsnpecan = callPackage ../applications/networking/instant-messengers/pidgin-plugins/msn-pecan { };
 
+  pidginmra = callPackage ../applications/networking/instant-messengers/pidgin-plugins/pidgin-mra { };
```
Comment (vcunat) [inline pkgs/top-level/all-packages.nix]: Better use `pidgin-mra` for new packages. It's preferable to use dashes, especially if they're used upstream (or in most distros). http://nixos.org/nixpkgs/manual/#sec-package-naming

File: pkgs/applications/networking/instant-messengers/pidgin-plugins/pidgin-mra/default.nix (hunk 1 above)
Comment (akaWolf) [inline pkgs/applications/networking/instant-messengers/pidgin-plugins/pidgin-mra/default.nix]: This line is needed. The builder have access to pkg-config. I guess, pkg-config is derived from pidgin.

File: pkgs/top-level/all-packages.nix (hunk 2 above)
Comment (akaWolf) [inline pkgs/top-level/all-packages.nix]: Ok.

File: pkgs/applications/networking/instant-messengers/pidgin-plugins/pidgin-mra/default.nix (hunk 1 above)
Comment (vcunat) [inline pkgs/applications/networking/instant-messengers/pidgin-plugins/pidgin-mra/default.nix]: I see, but better make it explicit.

**Pull request closed without merging**